front/app/components/ProjectCard/index.tsx

```diff
@@ -701,11 +701,13 @@ const ProjectCard = memo<Props>(
           {size !== 'large' && contentHeader}
 
           <ProjectImageContainer className={size}>
-            <ProjectImagePlaceholder>
-              <ProjectImagePlaceholderIcon name="project" />
-            </ProjectImagePlaceholder>
-
-            {imageUrl && <ProjectImage src={imageUrl} alt="" cover={true} />}
+            {imageUrl ? (
```

Comment:
The issue with the aria-labelledby was caused by the icon, but this placeholder should also not load when there's an image.